.github/workflows/main.yml

```diff
@@ -2,6 +2,22 @@ name: tests
 
 on: [push, pull_request]
 
+env:
+  FORCE_COLOR: 1  # Request colored output from CLI tools supporting it
+  MYPY_FORCE_COLOR: 1  # MyPy's color enforcement
+  PIP_DISABLE_PIP_VERSION_CHECK: 1
+  PIP_NO_PYTHON_VERSION_WARNING: 1
+  PIP_NO_WARN_SCRIPT_LOCATION: 1
+  PY_COLORS: 1  # Recognized by the `py` package, dependency of `pytest`
+  TOX_PARALLEL_NO_SPINNER: 1
+  TOX_TESTENV_PASSENV: >-  # Make tox-wrapped tools see color requests
```

Comment:
I imagine this env var is going to conflict or be overridden in some projects that use it for other purposes. I wonder if tox should support these by default. Have you explored that with the project?

Reply:
My understanding is that these could actually go into `tox.ini`.